rfcs/formats/assets/exchange.md

```diff
@@ -58,10 +68,11 @@ message AssetClaim {
 Since every claim is associated with a lock, the above structure needs a `lockMechanism` field just like the `AssetLock` structure does, with identical semantics. The `claimInfo` field is a serialized claim structure containing specific information about the claim. For the `HTLC` locking mechanism, the claim structure is as follows:
 ```protobuf
 message AssetClaimHTLC {
-  bytes hashPreimageBase64 = 1;
+  HashMechanism hashMechanism = 1;
+  bytes hashPreimageBase64 = 2;
 }
 ```
-An HTLC claim simply needs to specify the secret preimage (`hashPreimageBase64`) of the hash in the corresponding HTLC lock. It is encoded in Base64 for communication safety and portability.
+An HTLC claim simply needs to specify the secret preimage (`hashPreimageBase64`) of the hash in the corresponding HTLC lock. It is encoded in Base64 for communication safety and portability. The `hashMechanism` field assumes a value from the `HashMechanism` enumeration specified earlier, corresponding to the lock structure `AssetLockHTLC` defined earlier.
```

Comment:
", corresponding to the lock structure `AssetLockHTLC` defined earlier." --> ", and must match the `hashMechanism` value in the `AssetLockHTLC` object corresponding to this claim."